planq/assetlist.json

```diff
@@ -29,6 +29,46 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/planq/images/planq.svg"
         }
       ]
+    },
+    {
+      "description": "The token of Source Protocol bridged from BSC.",
+      "denom_units": [
+        {
+          "denom": "erc20/0x091F9A57A3F58d758b6572E9d41675918EAC7F09",
+          "exponent": 0
+        },
+        {
+          "denom": "srcx",
+          "exponent": 9
+        }
+      ],
+      "base": "erc20/0x091F9A57A3F58d758b6572E9d41675918EAC7F09",
+      "name": "Source Token",
+      "display": "srcx",
+      "coingecko_id": "source-protocol",
+      "symbol": "SRCX",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "binancesmartchain",
+            "base_denom": "0x454b90716a9435e7161a9aea5cf00e0acbe565ae",
+            "contract": "0x454b90716a9435e7161a9aea5cf00e0acbe565ae"
+          },
+          "chain": {
+            "contract": "0xC891aBa0b42818fb4c975Bf6461033c62BCE75ff"
+          },
+          "provider": "DeltaSwap.io"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/planq/images/srcx.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/planq/images/srcx.png"
+        }
```

Comment:
Can you please add the image only to: /non-cosmos/binancesmartchain/images/, and in this Planq-bridged asset, please use image_sync to point to the image in the bsc image folder